Sponsors.md

```diff
@@ -80,6 +79,18 @@ The list below describes the awareness benefits that our Logo sponsor receives i
 * “Thank you” mention in facebook & Twitter page post,
 * “Thank-you” message by organizers during Meetup (not by company/individual).
 
+## Equipment Sponsor
+As we are not close to the local community of Crete we try to live stream our monthly meetups and supply VoD through Youtube. FOr this reason we use equipment which mosto of the times is not suitable for our need.
+
+So we want some equipment to support even better our live stream capabilities such as two 1080p usb cameras and a deskptop micorphone.
+
+The list below describes the awareness benefits that our equipment sponsor receives in return for every 100 EUR offer:
+* leaflets (can be left on table at entrance),
+* online banner on DevStaff website (with back-link do-follow),
+* “Thank-you” message by organizers during Meetup (not by company),
+* Sponsor’s slide in DevStaff Welcome presentation,
+* “Thank you” mention in facebook page post.
+
```

Comment:
* "Thank you" mention on DevStaff's Twitter
* "Thank you" mention on DevStaff's LinkedIn
* Logo on DevStaff's Sponsors Slide in Live Stream

Reply:
Fixed by https://github.com/devstaff-crete/DevStaff-Heraklion/pull/413/commits/2b308bbb5a661f83a7deec7f2a009fb3a90f38fa

We don't maintain LinkedIn but we maintain a Twitter account!